src/main/java/org/folio/circulation/domain/CheckInProcessRecords.java

```diff
@@ -81,7 +86,8 @@ public CheckInProcessRecords withRequestQueue(RequestQueue requestQueue) {
       this.loan,
       requestQueue,
       this.checkInServicePoint,
-      firstRequest);
+      firstRequest,
+    this.loggedInUserId);
```

Comment:
GitHub shows the indentation is different, why is that the case?

Reply:
Done.